repair/row_level.cc

```diff
@@ -2290,32 +2290,45 @@ future<repair_update_system_table_response> repair_service::repair_update_system
 }
 
 future<repair_flush_hints_batchlog_response> repair_service::repair_flush_hints_batchlog_handler(gms::inet_address from, repair_flush_hints_batchlog_request req) {
-    rlogger.info("repair[{}]: Started to process repair_flush_hints_batchlog_request from node={}, target_nodes={}, hints_timeout={}s, batchlog_timeout={}s",
+    rlogger.info("repair[{}]: Started to process repair_flush_hints_batchlog_request from node={} target_nodes={} hints_timeout={}s batchlog_timeout={}s",
             req.repair_uuid, from, req.target_nodes, req.hints_timeout.count(), req.batchlog_timeout.count());
-    std::vector<gms::inet_address> target_nodes(req.target_nodes.begin(), req.target_nodes.end());
-    db::hints::sync_point sync_point = co_await _sp.local().create_hint_sync_point(std::move(target_nodes));
-    lowres_clock::time_point deadline = lowres_clock::now() + req.hints_timeout;
-    try {
-        co_await coroutine::all(
-            [this, &from, &req, &sync_point, &deadline] () -> future<> {
-                rlogger.info("repair[{}]: Started to flush hints for repair_flush_hints_batchlog_request from node={}, target_nodes={}", req.repair_uuid, from, req.target_nodes);
-                co_await _sp.local().wait_for_hint_sync_point(std::move(sync_point), deadline);
-                rlogger.info("repair[{}]: Finished to flush hints for repair_flush_hints_batchlog_request from node={}, target_hosts={}", req.repair_uuid, from, req.target_nodes);
-                co_return;
-            },
-            [this, &from, &req] () -> future<>  {
-                rlogger.info("repair[{}]: Started to flush batchlog for repair_flush_hints_batchlog_request from node={}, target_nodes={}", req.repair_uuid, from, req.target_nodes);
-                co_await _bm.local().do_batch_log_replay();
-                rlogger.info("repair[{}]: Finished to flush batchlog for repair_flush_hints_batchlog_request from node={}, target_nodes={}", req.repair_uuid, from, req.target_nodes);
-            }
-        );
-    } catch (...) {
-        rlogger.warn("repair[{}]: Failed to process repair_flush_hints_batchlog_request from node={}, target_hosts={}, {}",
-                req.repair_uuid, from, req.target_nodes, std::current_exception());
-        throw;
+    auto constexpr max_repair_hints_batchlog_flush_time = std::chrono::seconds(60);
```

Comment:
Please, make it configurable and LiveUpdate

Reply:
It is already a config in my local version. I did not push it yet.